### PR TITLE
Support travis

### PR DESCRIPTION
**Pull request by gforcada**

### CODE REVIEW

**Comment by gforcada** [comment]
@cjw296 tests are indeed passing, and coverage is at 98%. This is ready to merge.

Should the jenkins files still be kept? I see that they are making releases, I would suggest to move to [zest.releaser](https://pypi.python.org/pypi/zest.releaser) it's just *perfect* IMHO.

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[run]
+omit =
+  mailinglogger/tests/*.py
```
Comment (cjw296) [inline .coveragerc]: No, we want tests included. Measuring coverage on test files is an excellent way to check they're actually being run.
This has caught me out several times, so please remove this file.

File: MANIFEST.in [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+include *.txt
+include .coveragerc
+include buildout.cfg
+include jenkins
+include tox.ini
+recursive-include docs *.bat
+recursive-include docs *.py
+recursive-include docs *.txt
+recursive-include docs Makefile
+recursive-include mailinglogger *.txt
```
Comment (cjw296) [inline MANIFEST.in]: Why is this needed? See the mush project above, I think these would be a better way to go:
https://github.com/Simplistix/mush/blob/master/setup.py#L33
https://github.com/Simplistix/mush/blob/master/setup.py#L37

nb: setuptools-git ;-)

The setup.cfg in mush is probably also a good starting point...

File: requirements-dev.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+-e .
+nose
+nose_fixes
+testfixtures
+manuel
+nose-cov
```
Comment (cjw296) [inline requirements-dev.txt]: Nah, please just use an extra_requires['test'] as in mush, it's the pattern I use in all my projects...

**Comment by cjw296** [comment]
@gforcada - I prefer to do the releases from travis-ci, see the example in the Mush project I linked to.
I don't think I'll ever resurrect that Jenkins server, so please feel free to delete them and anything else that appears to be Jenkins-only.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+  - "2.7"
+install:
+  - pip install zc.buildout
+  - buildout
```
Comment (cjw296) [inline .travis.yml]: It's a fair bit to ask, but I'd love to lose the dependency on buildout.
Here's an example of the kind of .travis.yml I'd love to see, and indeed, the project structure I hope we can get to:

https://github.com/Simplistix/mush/blob/master/.travis.yml

**Comment by gforcada** [comment]
@cjw296 it took me a while to get back to it, but I think I addressed all your points

**Comment by cjw296** [comment]
Thanks!
